### ISSUE TITLE
can we convert the cDBG to GFA format with the current output files from spacegraphcats?

### ISSUE THREAD

**Issue by taylorreiter** [spacegraphcats/spacegraphcats#462]
I'm wondering if we can convert the current bcalm output by spacegraphcats to gfa. These are the current output files in `*_k31`:
```
bcalm.inputlist.txt  bcalm.unitigs.db      cdbg.gxt          contigs.mphf   reads.bgz.index
bcalm.log.txt        bcalm.unitigs.fa.sig  contigs.indices   contigs.sig
bcalm_to_gxt.log     bcalm.unitigs.pickle  contigs.info.csv  contigs.sizes
```

I used to do the conversion from the `*unitigs.fa` file but i think that became a `temp()` file that gets deleted.

**Comment by ctb** [comment]
yes, it's a `temp` file - https://github.com/spacegraphcats/spacegraphcats/blob/latest/spacegraphcats/conf/Snakefile#L168

and the header information is neither preserved nor output-table per https://github.com/spacegraphcats/spacegraphcats/blob/latest/spacegraphcats/cdbg/dump_contigs_db_to_fasta.py

so, we can't convert the _current_ output, no :(.

might be something to consider doing as part of https://github.com/spacegraphcats/spacegraphcats/pull/430 tho.

**Comment by taylorreiter** [comment]
ok great, thanks. I'm increasingly thinking of ways to better integrate with other people interested in metagenome assembly graphs, and I think gfa is a pretty standard format that wouldn't be a bad idea to either output or have a simple conversion script for. #430 looks exciting!
